mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -442,9 +442,19 @@ static FuncOp createCPUConvolution(ModuleOp &module, OpBuilder &builder,
   cpuConvBlock->push_back(inLayoutMemRefCastOp);
```

Comment:
This is a note to remember that the existing validation logic (based on whether cpu/gpu results are equal, and input of 1) is not good enough. To correctly validate the cpu/gpu results. There needs to be input of random numbers. Also, to compare float number close enough based on subtraction. 

If you decide to address that in a new review/issue I'd be fine with that.

Reply:
Agree. We should test on random inputs.